packages/react-component-playground/src/components/component-playground.jsx

```diff
@@ -11,8 +11,14 @@ var _ = require('lodash'),
     isSerializable = require('../lib/is-serializable.js').isSerializable,
     SplitPane = require('ubervu-react-split-pane'),
     localStorageLib = require('../lib/local-storage.js'),
+    CodeMirror = require('react-codemirror'),
     fuzzaldrinPlus = require('fuzzaldrin-plus');
 
+require('codemirror/lib/codemirror.css');
+require('codemirror/addon/fold/foldgutter.css');
+require('codemirror/theme/solarized.css');
```

Comment:
The thing is `react-codemirror` doesn't bundle any `codemirror` addons or styles, to keep the bundle size to a minimum.
JS `require` calls (for syntax highlight keyword detection and code folding) inject code into the global `codemirror` object.

However, webpack bundles `codemirror` twice, once for CP and once for `react-codemirror` and requires in CP are injecting code to the wrong instance. All I could come up with is move the `require` calls to `react-codemirror`, since we had the fork.
Maybe some dedupe plugin would work, i am not sure though...